web/app/themes/ppj/jobs-handler/import-jobs.php

```diff
@@ -63,25 +68,56 @@ function ppj_import_jobs($force_pull = false)
     $youth_custody_jobs = array();
     $youth_custody_locations = array();
 
-    $locations_json = file_get_contents(get_template_directory() . "/jobs-handler/locations.json");
+    //generated locations fro cpt
+    $locations_file = $upload_dir['basedir'] . "/job-feed/locations.json";
+    if(!file_exists($locations_file)){
+
+        //theme backup locations
+        $locations_file = get_template_directory() . "/jobs-handler/locations.json";
+
```

Comment:
Is this in case there are no entered locations? Just trying to work out why there is a jobs-handler and and job-feed , is one write and one read?